tools/build_file_generator/ThirdPartyDepsTool.java

```diff
@@ -45,34 +35,15 @@ public static void main(String[] args) throws IOException {
     ThirdPartyDepsToolComponent component =
         DaggerThirdPartyDepsTool_ThirdPartyDepsToolComponent.create();
 
-    createZip(component.getFileUtils(), component.getThirdPartyDepsAnalyzer().getThirdPartyDeps());
-  }
-
-  private static void createZip(FileUtils fileUtils, ThirdPartyDeps thirdPartyDeps)
-      throws IOException {
-    final String prototxtFile = "third_party_deps.prototxt";
-    String absBuildFileGeneratorDirPath =
-        fileUtils.joinPaths(fileUtils.getCurrentWorkingDirectory(), "tools/build_file_generator");
-
-    Map<String, String> env = new HashMap<>();
-    env.put("create", "true");
-
-    URI uri =
-        URI.create(
-            String.format(
-                "jar:file:%s",
-                fileUtils.joinPaths(absBuildFileGeneratorDirPath, "third_party_deps.zip")));
-    try (FileSystem zipfs = FileSystems.newFileSystem(uri, env)) {
-      fileUtils.writePrototxtUnchecked(
-          thirdPartyDeps, fileUtils.joinPaths(absBuildFileGeneratorDirPath, prototxtFile));
-      Path prototxtFilePath =
-          Paths.get(fileUtils.joinPaths(absBuildFileGeneratorDirPath, prototxtFile));
-      Files.copy(
-          prototxtFilePath, zipfs.getPath("/" + prototxtFile), StandardCopyOption.REPLACE_EXISTING);
-    } finally {
-      fileUtils.deleteFileOrDirectoryIfExists(
-          fileUtils.joinPaths(absBuildFileGeneratorDirPath, prototxtFile));
-    }
+    FileUtils fileUtils = component.getFileUtils();
+    component
+        .getFileUtils()
+        .writePrototxtToZip(
+            component.getThirdPartyDepsAnalyzer().getThirdPartyDeps(),
+            fileUtils.joinPaths(
+                fileUtils.getCurrentWorkingDirectory(),
+                "tools/build_file_generator/third_party_deps.zip"),
```

Comment:
Let's make these 2 strings `static final String`?
People looking at this tool class might want to see what the files are called, so doing that will put them higher up and more prominently in the class.

Reply:
Done